src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/ErrorMsg.java

```diff
@@ -17,14 +20,14 @@
     // region constructors
     @SuppressWarnings("squid:S00107") // number of params
     public ErrorMsg(
-        String id,
-        Long eventId,
-        String zipFileName,
-        String jurisdiction,
-        String poBox,
-        String documentControlNumber,
-        ErrorCode errorCode,
-        String errorDescription
+        @JsonProperty(value = "id", required = true) String id,
```

Comment:
Can we have a test which shows the exception we receive when required fields are not passed and service is able to handle it ?